src/utils/getLatestPackage.ts

```diff
@@ -0,0 +1,36 @@
+import { PubApiModel } from "../model/pubApiModel"
+import { Scalar, Pair } from "yaml/types"
+import axios from "axios"
+import semverDiff from "semver-diff"
+
+/**
+ * Returns the Latest version that the package can 
+ * automatically update to following semver.
+ */
+async function getLatestPackage(dependency: Pair): Promise<PubApiModel.Version | undefined> {
+    const dependencyKey: Scalar = dependency.key
+    const dependencyValue: Scalar = dependency.value
+
+    let dependencyName = dependencyKey.value
```

Comment:
why you reassign dependencyKey.value to another variable?
1. Can't you use dependencyValue declared above?
2. Why a "let"? You con't reassign it